docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,44 +1,99 @@
-one or more of your indices. That's it. That's all you need to configure to start using Kibana. You can
-add index patterns at any time from the <<settings-create-pattern,Management tab>>.
+To start working with your data in {kib}, you can:
 
-TIP: By default, Kibana connects to the Elasticsearch instance running on `localhost`. To connect to a
-different Elasticsearch instance, modify the Elasticsearch URL in the `kibana.yml` configuration file and
-restart Kibana. For information about using Kibana with your production nodes, see <<production>>.
+* Use the File Data Visualizer or GeoJSON Upload feature to quickly load
+data into {es}.
 
-To configure the Elasticsearch indices you want to access with Kibana:
+* Follow one of the Add Data tutorials to set up a Beats module to index logs,
+metrics, events, or application data.
 
-. Point your browser at port 5601 to access the Kibana UI. For example, `localhost:5601` or
-`http://YOURDOMAIN.com:5601`.
-+
-image:images/Start-Page.png[Kibana start page]
-+
-. Specify an index pattern that matches the name of one or more of your Elasticsearch indices. The pattern
-can include an asterisk (*) to matches zero or more characters in an index's name. When filling out your
-index pattern, any matched indices will be displayed.
-. Click *Next Step* to select the index field that contains the timestamp you want to use to perform time-based
-comparisons. Kibana reads the index mapping to list all of the fields that contain a timestamp. If your
-index doesn't have time-based data, choose *I don't want to use the Time Filter* option.
-+
-. Click *Create index pattern* to add the index pattern. This first pattern is automatically configured as the default.
-When you have more than one index pattern, you can designate which one to use as the default by clicking
-on the star icon above the index pattern title from *Management > Index Patterns*.
+* Connect {kib} with existing {es} indices.
 
-All done! Kibana is now connected to your Elasticsearch data. Kibana displays a read-only list of fields
-configured for the matching index.
+{kib} also provides <<get-data-in, sample data sets>> for you to use.
 
 [float]
-[[explore]]
-=== Start Exploring your Data!
-You're ready to dive in to your data:
+[[upload-data-kibana]]
+=== Use the File Data Visualizer
+
+To visualize data in a CSV,  JSON, or log file, you can
+upload it using the File Data Visualizer.  On the home page,
+click *Import a CSV, NDSON, or log file*, and then drag your file into the
+File Data Visualizer. You can upload a file up to 100 MB.
+
+[role="screenshot"]
+image::images/add-data-fv.png[File Data Visualizer]
+
+The File Data Visualizer uses the {ref}/ml-find-file-structure.html[find_file_structure API] to analyze
+the uploaded file and to suggest ingest pipelines and mappings for your data.
+
+NOTE: This feature is not intended for use as part of a
```

Comment:
I'm not sure what this note means. Why wouldn't I be able to use this feature repeatedly? 

The second part, `rather for the initial exploration of your data` sounds like part of the value prop.